src/client/Microsoft.Identity.Client/WwwAuthenticateParameters.cs

```diff
@@ -50,6 +58,17 @@ public class WwwAuthenticateParameters
         /// </summary>
         public string Error { get; set; }
 
+        /// <summary>
+        /// AuthScheme.
+        /// See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/WWW-Authenticate#syntax for more details
+        /// </summary>
+        public string AuthScheme { get; private set; }
+
+        /// <summary>
+        /// Pop Nonce. This is acquired from with the POP WWW-Authenticate header.
+        /// </summary>
+        public string PopNonce { get; private set; }
+
         /// <summary>
         /// Return the <see cref="RawParameters"/> of key <paramref name="key"/>.
```

Comment:
I think we should exclude the resource / scope from the raw parameters as well. @jmprieur - thoughts?